applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -44,8 +40,10 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_USB_STATE_LOG_LEVEL);
 #define USB_SUBSCRIBER_PIPELINE_SIZE 0x01
 #define USB_SUBSCRIBER_REPORT_MAX    0x01
 
-#if DT_PROP(DT_NODELABEL(usbd), num_in_endpoints) < (CONFIG_USB_HID_DEVICE_COUNT + 1)
-  #error Too few USB IN Endpoints enabled. Modify dts.overlay file.
+/* Not used for USB next stack. */
+#ifdef CONFIG_DESKTOP_USB_NEXT
+	#define CONFIG_USB_HID_DEVICE_NAME ""
+	#define CONFIG_DESKTOP_USB_HID_PROTOCOL_CODE -1
 #endif
```

Comment:
is it necessary to overwrite those kconfigs?

Reply:
Yes, otherwise compilation fails because of an undefined symbol